config.yaml

```diff
@@ -1,8 +1,5 @@
-extensions:
-  zpages:
-    endpoint: 0.0.0.0:55679
-
 receivers:
```

Comment:
A couple of people suggested that it would be nice to keep these separate. This is the one that user's are expected to edit, whereas as the other one is for things they shouldn't change, or for advanced users to change if they really want. The conern is that having a 300 line config would be confusing to people.

Having said that, maybe two configs is confusing as well, especially since it deviates from regular Collector behaviour. I will think about it a bit more.

Reply:
Reverted for now, mostly because I don't want to have to update all the packaging code to support multiple config files, but have checked this code into another branch and may come back to it in the future